integration-tests/src/test/java/io/quarkiverse/argocd/it/ArgoConfigurator.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkiverse.argocd.it;
+
+import lombok.*;
+
+@Builder
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class ArgoConfigurator {
+    @Builder.Default
+    private String projectName = "default";
+
+    @Builder.Default
```

Comment:
Neatpicling here, but it would be nice to Import `Default`.

Reply:
Fixed: daf3420